apps/ima/views/form-submitted.html

```diff
@@ -1,7 +1,7 @@
 <title>{{#t}}pages.form-submitted.confirmed{{/t}} – GOV.UK</title>
 
-{{<partials-page}} {{$page-content}} <!-- alert-complete is the HOF supported class currently, should be using the
-  govuk-styles -->
+{{<partials-page}} {{$page-content}}
```

Comment:
put `{{$page-content}}` on the next line

Reply:
{{$page-content}} moved to next line